### DISCUSSION TITLE
How to change the allowOther: Other

### DISCUSSION THREAD

**Discussion by danielpetinga** [ditdot-dev/vue-flow-form#287] [General] [answered]
Hello guys,

I'm looking for a way to customize the label that is displayed when I use the allowOther option. Instead of "Other", I would like to enter a different label. Would anyone know how to do this?

I thank you for your help!

**Comment by spinn** [comment, marked as answer]
You can do this by overriding the default label by providing a custom language object:

https://www.ditdot.hr/en/docs/vue-flow-form-guide#multilanguage

**Reply by danielpetinga** [reply]
It worked, thanks!
